### ISSUE TITLE
(Linux) Lockup on Scene Change : From SPACECENTER to FLIGHT

### ISSUE THREAD

**Issue by dimension128** [linuxgurugamer/ksp-advanced-flybywire#6]
With AFBW installed, the black loading screen shows up, and never goes away after clicking launch.
[KSP.log](https://github.com/linuxgurugamer/ksp-advanced-flybywire/files/2068858/KSP.log)

**Comment by linuxgurugamer** [comment]
From the page which says what is needed:

https://forum.kerbalspaceprogram.com/index.php?/topic/83212-how-to-get-support-read-first/

**NOTE: These logs are not the same as KSP.log, which lacks valuable data. Do not upload KSP.log; do upload output_log.txt / Player.log**

**Comment by linuxgurugamer** [comment]
this is a support request, please post in the forums first before raising an issue.  That can give other people a chance to help as well.

**Comment by dimension128** [comment]
Thank you for the information. I was confused, as I've not been involved with ksp for more than a year, and things have changed. Many of the old forum threads I had bookmarked are gone, my status is Newbie with just one post, and the links in the readme on this github repo are also dead. I just remembered that AFBW is something I wanted. So I fired up ckan and installed it, then went back to ckan when it wasn't working, and clicked the github link. (Because of the aforementioned frustration with the forums I did skip that link).

I have posted, following the format described in the instructions you provided... quite a while ago. It's still awaiting moderator approval. If you wish to look at the log without waiting for the forum, it can be found  [here](https://www.dropbox.com/s/d8v33a837fgm07l/With_AFBW_Player.log?dl=0).
Either way, thank you for what you are doing. I wish Squad themselves would properly support the systems they sell this product for, but it seems like most linux games require people to clean up the messes the paid devs leave for us. I don't know why something like Xplane11 can work perfectly with all of my joysticks, but ksp, having been out of beta for a long time, still can't.
